CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 * None.
 
 ### Compatibility
-* Fileformat: Generates files with format v23. Reads and automatically upgrade from fileformat v5.
+* Fileformat: Generates files with format v24. Reads and automatically upgrade from fileformat v10.
```

Comment:
We should make this more discoverable. It's a kind of "Breaking change" compared to previous bahavior. So an entry there would be useful.